QuantLib/ql/models/calibrationhelper.hpp

```diff
@@ -64,7 +71,7 @@ namespace QuantLib {
         virtual Real modelValue() const = 0;
 
         //! returns the error resulting from the model valuation
-        virtual Real calibrationError();
+        Real calibrationError() const;
```

Comment:
I agree that this should have been `const` from the beginning, but making it `const` now can break code.